repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -679,6 +679,8 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
       akkaServiceClient = akkaServiceClientFactory.newAkkaServiceClient(keystoneV2Config.getIdentityService.getConnectionPoolId)
       akkaServiceClientOld.foreach(_.destroy())
 
+      ignoredRoles = keystoneV2Config.getIgnoredRoles.split(' ').to[Set]
```

Comment:
This ties into my comment about using the XSD `list` type. I believe that type collapses whitespace when parsing, whereas this code does not. Thus, a user could end up with empty strings as ignored roles if a user was expecting the XSD `list` semantics.